src/databases/Silo/avtSiloOptions.C

```diff
@@ -144,5 +145,26 @@ GetSiloWriteOptions(void)
     rv->SetBool(SILO_WROPT_CKSUMS, false);
     rv->SetString(SILO_WROPT_COMPRESSION, "");
 
+    char helpStr[512];
```

Comment:
Add modification comment to this function for the help string added.

Reply:
Ok, got that